tests/mock.c

```diff
@@ -180,6 +189,14 @@ int ioctl(int fd, unsigned long request, ...)
 				FPGA_MSG("unexpected port ID %u", port_release->port_id);
 				goto out_EINVAL;
 			}
+
+			// Returns error when Error injection enabled
+			if (gEnableErrInj) {
+				retval = EINVAL;
+				errno = EINVAL;
+				break;
+			}
+
```

Comment:
Can all of these be simplified by moving this check up to around line 168?
If the flag is set, then we can simply call
```C
goto out_EINVAL;
```